superset-frontend/spec/javascripts/explore/utils_spec.jsx

```diff
@@ -200,6 +201,25 @@ describe('exploreUtils', () => {
     });
   });
 
+  describe('columnCount', () => {
+    it('divides samples data into pages of 20', () => {
+      const pageSize = columnCount(500);
+      expect(pageSize).toEqual(20);
+    });
+    it('divides samples data into pages of 20', () => {
+      const pageSize = columnCount(0);
+      expect(pageSize).toEqual(50);
+    });
+    it('divides samples data into pages of 20', () => {
+      const pageSize = columnCount(1);
+      expect(pageSize).toEqual(10000);
+    });
+    it('divides samples data into pages of 20', () => {
+      const pageSize = columnCount(1000000);
+      expect(pageSize).toEqual(5);
+    });
+  });
```

Comment:
This probably comes down to preference, but I think we could get by with a single `it` here, just doing multiple `expect(columnCount(xxx)).toEqual(yyy);` in sequence, as this test is pretty simple. Just make sure the test description is clear on what's being tested. But if we do keep separate `it`s here, make sure to update the descriptions appropriately (all have a reference to 20, which only applies to the first case).

Reply:
yep, forgot to change the description. Will move it under one `it`